uwrt_mars_rover_bringup/scripts/bringup_fake_can.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
nit: but i think virtual can is a better description than fake can

Reply:
I called it fake can because I'm breaking the convention that virtual can interfaces are named as vcan0, etc. This is just of ease of operability with the other stuff that directly uses the interface name. I'll clean this up in a follow up later.

Reply:
#146